tests/containers/pydantic_schemas/podman_info.py

```diff
@@ -0,0 +1,466 @@
+from __future__ import annotations
```

Comment:
This comes from somewhere or it's written by you, Jiri? If the source is somewhere else, maybe we should link the source for eventual reference? If not, then I'm just fine.

Reply:
This is created by smart tooling that was given examples of the json data and it then generates pydantic schema for it. So it's all my work.